packages/components/hub/HubLanding.tsx

```diff
@@ -33,12 +33,17 @@ export const HubLanding: React.FC = () => {
   const networkId = useSelectedNetworkId();
   const banners = useBanners(networkId);
   const banner = banners?.length ? banners[0] : undefined;
+  const postId = banner?.url
+    ? banner.url.split("/")[banner.url.split("/").length - 1]
+    : "";
 
   return (
     <View style={{ alignItems: "center", width: "100%" }}>
       <View style={{ flex: 1 }}>
         {!!banner && (
-          <Link to={banner?.url || ""}>
+          <CustomPressable
+            onPress={() => navigation.navigate("FeedPostView", { id: postId })}
+          >
```

Comment:
why are you changing this behavior? the banner url can be an external link

Reply:
Currently the banner url is /feed/post/tori-bcd1253e-8b1a-4268-b3d8-59a1541d3fe2 (not external one). So I thought the only url we get is for feedpostview. If it can also be the external link  then, I will adjust that one. I used CustomPressable because the Link component previously used was causing the image to automatically grow bigger.

Reply:
Fixed the problem with original Link component